pom.xml

```diff
@@ -78,28 +78,31 @@
 
   <properties>
     <maven-jar-plugin.version>3.2.0</maven-jar-plugin.version>
+    <maven-dependency-plugin.version>3.3.0</maven-dependency-plugin.version>
     <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
     <maven-failsafe-plugin.version>2.22.2</maven-failsafe-plugin.version>
     <maven-shade-plugin.version>3.2.4</maven-shade-plugin.version>
     <maven-javadoc-plugin.version>3.1.1</maven-javadoc-plugin.version>
-    <maven-compiler-plugin.version>3.8.0</maven-compiler-plugin.version>
+    <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
     <maven-deploy-plugin.version>2.4</maven-deploy-plugin.version>
     <genjavadoc-plugin.version>0.15</genjavadoc-plugin.version>
     <build-helper-maven-plugin.version>1.7</build-helper-maven-plugin.version>
     <maven-enforcer-plugin.version>3.0.0-M1</maven-enforcer-plugin.version>
     <maven-docker-plugin.version>0.37.0</maven-docker-plugin.version>
 
     <java.version>1.8</java.version>
-    <fasterxml.version>2.6.7</fasterxml.version>
-    <fasterxml.jackson.databind.version>2.6.7.3</fasterxml.jackson.databind.version>
-    <fasterxml.jackson.module.scala.version>2.6.7.1</fasterxml.jackson.module.scala.version>
-    <fasterxml.jackson.dataformat.yaml.version>2.7.4</fasterxml.jackson.dataformat.yaml.version>
+    <fasterxml.version>${fasterxml.spark3.version}</fasterxml.version>
+    <fasterxml.jackson.databind.version>${fasterxml.spark3.version}</fasterxml.jackson.databind.version>
+    <fasterxml.jackson.module.scala.version>${fasterxml.spark3.version}</fasterxml.jackson.module.scala.version>
+    <fasterxml.jackson.dataformat.yaml.version>${fasterxml.spark3.version}</fasterxml.jackson.dataformat.yaml.version>
     <fasterxml.spark3.version>2.10.0</fasterxml.spark3.version>
-    <kafka.version>2.0.0</kafka.version>
-    <kafka.spark3.version>2.4.1</kafka.spark3.version>
+    <kafka.version>${kafka.spark3.version}</kafka.version>
+    <kafka.spark2.version>2.0.0</kafka.spark2.version>
+    <kafka.spark3.version>2.8.0</kafka.spark3.version>
     <pulsar.version>2.8.1</pulsar.version>
     <confluent.version>5.3.4</confluent.version>
     <glassfish.version>2.17</glassfish.version>
+    <parquet.version>1.12.1</parquet.version>
```

Comment:
This should be 1.12.2 right ? Atleast thats what I see being used on the master already for Spark 3.

Reply:
Yes in the root pom seems i might have made a typo, however currently in the pr the spark3 profile has the correct parquet version.
 ```
<profile>
      <id>spark3</id>
      <properties>
        <spark3.version>3.2.1</spark3.version>
        <spark.version>${spark3.version}</spark.version>
        <sparkbundle.version>3</sparkbundle.version>
        <scala.version>${scala12.version}</scala.version>
        <scala.binary.version>2.12</scala.binary.version>
        <hudi.spark.module>hudi-spark3</hudi.spark.module>
        <hudi.spark.common.module>hudi-spark3-common</hudi.spark.common.module>
        <scalatest.version>${scalatest.spark3.version}</scalatest.version>
        <kafka.version>${kafka.spark3.version}</kafka.version>
        <parquet.version>1.12.2</parquet.version>
```
as well as the spark 2 profile
```
  <profile>
      <id>spark2</id>
      <properties>
        <spark.version>${spark2.version}</spark.version>
        <sparkbundle.version>${spark2.bundle.version}</sparkbundle.version>
        <scala.version>${scala11.version}</scala.version>
        <scala.binary.version>${spark2.scala.binary.version}</scala.binary.version>
        <hudi.spark.module>hudi-spark2</hudi.spark.module>
        <hudi.spark.common.module>hudi-spark2-common</hudi.spark.common.module>
        <scalatest.version>3.0.1</scalatest.version>
        <kafka.version>2.0.0</kafka.version>
        <parquet.version>1.10.1</parquet.version>
```